README.md

```diff
@@ -24,6 +25,7 @@
 </p>
 
 ## Table of Contents
+
 - [Table of Contents](#table-of-contents)
```

Comment:
I just noticed the table of contents isn't reflecting these new changes. We should probably update it 👍.

Reply:
Fixed